src/current/_includes/releases/v22.2/v22.2.17.md

```diff
@@ -0,0 +1,33 @@
+## v22.2.17
+
+<!-- Release Date: November XX, 2023 -->
+
+{{ site.data.alerts.callout_info }}
+v22.2.17 is currently available only on select CockroachDB {{ site.data.products.dedicated }} clusters. If you wish to upgrade your {{ site.data.products.dedicated }} cluster to v22.2.17, please submit a [support request](https://support.cockroachlabs.com/hc/en-us/requests/new).
+
+This release is currently not available for CockroachDB {{ site.data.products.serverless }} or {{ site.data.products.core }} customers.
+{{ site.data.alerts.end }}
+
+<!-- {% include releases/release-downloads-docker-image.md release=include.release %} -->
+
+<h3 id="v22-2-17-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could prevent RESTORE from working if it was performed during a cluster upgrade. [#112757][#112757]
+- Queries with the `st_union` aggregate function could produce incorrect results in some cases due to the query optimizer performing invalid optimizations. This bug has been present since the `st_union` function was introduced in version 20.2.0. [#112796][#112796]
+- CockroachDB previously could incorrectly evaluate lookup and index joins into tables with at least 3 column families in some cases (either "non-nullable column with no value" internal error would occur or the query would return incorrect results), and this is now fixed. The bug was introduced in 22.2. [#113109][#113109]
+- Fixed a rare internal error in the optimizer that has existed since before version 22.1, which could occur while enforcing orderings between SQL operators. [#113175][#113175]
```

Comment:
Link optimizer doc. Also, "since before v22.1" sounds weird to me, since that implies we can cite a specific version "since at least vXX.X"?

```suggestion
- Fixed a rare internal error in the optimizer that could occur while enforcing orderings between SQL operators. This bug had existed since before v22.1. [#113175][#113175]
```

Reply:
Ya it's weird.  Everything before v22.1 is now unsupported though.  I would guess the subtext here is that our automated testing doesn't cover those older unsupported versions so we know it's been "at least" since then

Thank you for the edit too! will add the link